chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/ChatManagerFixtures.kt

```diff
@@ -55,25 +54,25 @@ class TestChatkitDependencies(
     override val appHooks = NoAppHooks()
 }
 
-val SynchronousCurrentUser.generalRoom
-    get() = rooms.find { it.name == Rooms.GENERAL } ?: error("Could not find room general")
-
-private val managers = ConcurrentLinkedQueue<SynchronousChatManager>()
-
-fun chatFor(userName: String) = SynchronousChatManager(
-        instanceLocator = INSTANCE_LOCATOR,
-        userId = userName,
-        dependencies = TestChatkitDependencies(
-                tokenProvider = TestTokenProvider(INSTANCE_ID, userName, AUTH_KEY_ID, AUTH_KEY_SECRET)
-        )
-).also { managers += it }
-
-fun closeChatManagers() {
-    for (manager in managers) {
-        manager.close()
-    }
-    managers.clear()
-}
+// val SynchronousCurrentUser.generalRoom
```

Comment:
what's the motivation for keeping those lines commented out? I think they should be also removed

Reply:
I left things that i wasn't sure if we wanted to keep commented out - happy to remove :)

Reply:
Have removed 👍